src/lib/rest/rest.cpp

```diff
@@ -922,14 +924,19 @@ static ApiVersion apiVersionGet(const char* path)
     return V2;
   }
 
-  // Different from v2, v1 is case-insensitive (see case/2057 test)
```

Comment:
But note we will not need the info regarding ngsi9 or ngsi10 in any place, any taking into account the "extended set" of counter now in `#if 0`. So it makes sense to throw away it.